RiotSharpTest/RiotApiTest.cs

```diff
@@ -242,6 +243,30 @@ public void GetChampionAsync_Test()
 
         [TestMethod]
         [TestCategory("RiotApi")]
+        public void GetCurrentGame_Test()
+        {
+            if (idWithCurrentgame != 0) {
+                var currentGame = api.GetCurrentGame(Platform.NA1, idWithCurrentgame);
+
+                Assert.IsNotNull(currentGame);
+                Assert.IsTrue(currentGame.Participants.Any(p => p.SummonerId == idWithCurrentgame));
+            }
+        }
+
+        [TestMethod]
+        [TestCategory("RiotApi"), TestCategory("Async")]
+        public void GetCurrentGameAsync_Test()
+        {
+            if (idWithCurrentgame != 0) {
+                var currentGame = api.GetCurrentGameAsync(Platform.NA1, idWithCurrentgame);
+
+                Assert.IsNotNull(currentGame.Result);
+                Assert.IsTrue(currentGame.Result.Participants.Any(p => p.SummonerId == idWithCurrentgame));
+            }
+        }
+
+        [TestMethod]
+        [TestCategory("RiotApi")]
```

Comment:
Unfortunately, I don't think it's worth adding unit tests for the current game endpoint as we'll never be able to run them consistently.